internal/ingest/ingest.go

```diff
@@ -698,9 +698,17 @@ func (ing *Ingester) ingestWorkerLogic(msg toWorkerMsg) {
 	}
 
 	log.Infow("Running worker on ad stack", "headAdCid", msg.adInfos[0].cid, "publisher", msg.publisher, "numAdsToProcess", splitAtIndex)
+	var count int
 	for i := splitAtIndex - 1; i >= 0; i-- {
 		// Note that we are iterating backwards here. Earliest to newest.
 		ai := msg.adInfos[i]
+
+		count++
+		log.Infow("Processing advertisement",
+			"adCid", ai.cid,
+			"publisher", msg.publisher,
+			"progress", fmt.Sprintf("%d of %d", count, splitAtIndex))
```

Comment:
this will be a lot of log lines. are we worried this will be too spamy? seems maybe `Debug` level?

Reply:
It was adding fields to the existing "Processing advertisement" log entry, of which there is one per advertisement.  I think this should be at info level as it denotes the start ad processing.  I thing there are log entries for individual entries that I can move to debug.